schema/aggregations/aggregation_count.js

```diff
@@ -1,3 +1,6 @@
+// @flow
+import type { GraphQLFieldConfig } from 'graphql';
+
```

Comment:
the `// @flow` means this file should run with flow checking, which is pulled from the graphql module - see this [tweet](https://twitter.com/orta/status/808695513533059072)